drivers/counter/counter_esp32.c

```diff
@@ -238,7 +238,7 @@ static const struct counter_driver_api counter_api = {
 
 static void counter_esp32_isr(void *arg)
```

Comment:
```suggestion
static void counter_esp32_isr(const struct device *dev)
```

Reply:
This one could not be found by the script, as it does note use IRQ_CONNECT macro.